prg_test_helpers.go

```diff
@@ -34,32 +33,30 @@ func GetNextUInt64NewPRGWithSalt(
 	seed []byte,
 	salt []byte,
 ) (uint64, error) {
+	t.Helper()
```

Comment:
always use t.Helper() in test helpers, it makes the stacktrace a lot easier to read.